yasa/evaluation.py

```diff
@@ -0,0 +1,649 @@
+        ax : :py:class:`matplotlib.axes.Axes`
+            Axis on which to draw the plot, optional.
+        **kwargs : key, value pairs
+            Additional keyword arguments are passed to the :py:func:`matplotlib.pyplot.plot` call.
+
+        Returns
+        -------
+        ax : :py:class:`matplotlib.axes.Axes`
+            Matplotlib Axes
+        """
+        raise NotImplementedError("Requires probability/confidence values.")
+
+class SleepStatsEvaluation:
+    """
+    Evaluate agreement between two measurement devices by comparing summary sleep statistics across
+    multiple participants or sessions.
+
+    For example, the reference device might be PSG and the test device might be a wearable device.
+
+    Parameters
+    ----------
+    data : :py:class:`pandas.DataFrame`
+        A pandas dataframe with sleep statistics from two different
+        devices for multiple subjects
+    reference : str
+        Name of column containing the reference device sleep statistics.
+    test : str
+        Name of column containing the test device sleep statistics.
+    subject : str
+        Name of column containing the subject ID.
+    statistic : str
+        Name of column containing the name of the sleep statistics.
+
+    Notes
+    -----
+    Many steps here are modeled after guidelines proposed in Menghini et al., 2021 [Menghini2021]_.
+    See https://sri-human-sleep.github.io/sleep-trackers-performance/AnalyticalPipeline_v1.0.0.html
+
+    References
+    ----------
+    .. [Menghini2021] Menghini, L., Cellini, N., Goldstone, A., Baker, F. C., & de Zambotti, M.
+                      (2021). A standardized framework for testing the performance of sleep-tracking
+                       technology: step-by-step guidelines and open-source code. Sleep, 44(2),
+                       zsaa170. https://doi.org/10.1093/sleep/zsaa170
+
+    Examples
+    --------
+    >>> import pandas as pd
+    >>> import yasa
+    >>>
+    >>> # For this example, generate a fake dataset of sleep statistics from two different raters
+    >>> data = []
+    >>> for i in range(1, 21):
+    >>>     hypA = yasa.simulate_hypnogram(tib=600, seed=i)
+    >>>     hypB = hypA.simulate_similar(seed=i)
+    >>>     data.append({"subject": f"sub-{i:03d}", "rater": "RaterA"} | hypA.sleep_statistics())
+    >>>     data.append({"subject": f"sub-{i:03d}", "rater": "RaterB"} | hypB.sleep_statistics())
+    >>> df = (pd.json_normalize(data)
+    >>>     .melt(id_vars=["subject", "rater"], var_name="sstat", value_name="score")
+    >>>     .pivot(index=["subject", "sstat"], columns="rater", values="score")
+    >>>     .reset_index().rename_axis(None, axis=1)
+    >>>     .query("sstat.isin(['SE', 'TST', 'SOL', 'WASO', '%N1', '%N2', '%N3', '%REM'])")
+    >>> )
+    >>>
+    >>> sse = yasa.SleepStatsEvaluation(
+    >>>     data=df, reference="RaterA", test="RaterB", subject="subject", statistic="sstat"
+    >>> )
+    >>>
+    >>> sse.summary(descriptives=False)
```

Comment:
I would start by printing the summary with descriptive statistics

Reply:
Agreed. Now that you mention it, I wonder if we even need the ability to set `descriptives=False` at all?

Reply:
Probably not